### ISSUE TITLE
[iOS] images show blurred

### ISSUE THREAD

**Issue by Burgov** [NativeScript/nativescript-imagepicker#195]
All images show blurred in iOS, exactly as described here: https://github.com/questbeat/QBImagePicker/issues/86 (except I'm not doing multiselect, not sure if it's relevant). Since this lib is using that library, I suspect it's the same issue, but I don't understand the described workaround over there.

All iOS phones are affected (didn't try Android). 
NS 4.0.2
plugin latest version.

How can I resolve this problem?

**Comment by Burgov** [comment]
Possibly also related: https://github.com/questbeat/QBImagePicker/issues/115

**Comment by lini** [comment]
Hi,
Can you give me some more information about this issue so I can try to reproduce it with the image picker plugin? I tried using an iOS 11 device (real and simulator) with the demo app from this repository and all images looked sharp as thumbnails during the selection. If you can isolate the issue in a sample project, please send a link. Also, specify the iOS version/device model where you observe the blurred thumbnails.

**Comment by Burgov** [comment]
Hi @lini, thanks for looking into this. 

This issue appears to be hitting all iPhone devices. Tested with a range between 6 and 8 (didn't try X)

I'll try to see if i can isolate the issue or otherwise try and organize access to our repo so you can run that (will be next week)

**Comment by anki247** [comment]
Packege.json
"@angular/core": "~5.2.0",
"nativescript-imagepicker": "^6.0.2",

**Usage:**
````typescript
loadPhoto(callback: (selection: ImageAsset[]) => void) {
    this.context = imagepicker.create({
      mode: 'single', // use "multiple" for multiple selection
      mediaType: imagepicker.ImagePickerMediaType.Image
    });

    this.context
      .authorize()
      .then(() => this.context.present())
      .then(callback)
      .catch((ex) => {
        ...
      });
  }
````


**Blurred**
![blurred](https://user-images.githubusercontent.com/3366211/42418719-381bfe04-82a6-11e8-9f39-2f10ed1184a7.png)


What I also noticed is, that if I would reuse the `this.context` without creation on the second time, then the images are sharp, but then you can not navigate back to root album

**Comment by anki247** [comment]
Finally I was able to locate the problem, but I still have no idea how to solve it.
The above described blurred effect appears if you using the RadSideDrawer plugin 😕 
`"nativescript-ui-sidedrawer": "^4.1.1",`
I really can not explain why. I created a demo repo [here](https://github.com/anki247/image-picker-blurred-demo) to illustrate.
In `app/app.component.ts` you can switch between the two template version, the one without the side-drawer works as expected. (Images are sharp)
Is there any workaround for that? I really need to solve this issue as soon as possible.
tnx in advance

**Comment by anki247** [comment]
At least for me I found the solution/mistake I made.
You can check it out [here](https://github.com/telerik/nativescript-ui-feedback/issues/757#issuecomment-408713031)

**Comment by JakubPawlak** [comment]
I have the same problem in iOS. In my case, I've found that the problem is related to TabView and multiple page router outlets (I'm using Angular) in the root app component. Here is the minimal project on git I made to reproduce the issue.
https://github.com/JakubPawlak/nativescript-blurry-imagepicker.git
If there is no page router outlet (just comment it in app.component.html), only TabViews, a viewController of undefined error is thrown, otherwise plugin is started properly but images in the gallery are blurred.
